packages/portal-api/src/ts/handler.ts

```diff
@@ -0,0 +1,34 @@
+import { addressUtils, logUtils } from '@0xproject/utils';
+import * as express from 'express';
+import * as _ from 'lodash';
+import { Op } from 'sequelize';
+
+import { models } from './models';
+
+export const handler = {
+    async getPrices(req: express.Request, res: express.Response) {
+        const tokensParam = req.query.tokens; // should be a string of comma delimited token symbols
+        const tokenSymbols = tokensParam.split(',');
```

Comment:
Think we need to sanitise this input some more. Make sure it's alphabetical + `,`